### DISCUSSION TITLE
My ssd1963 lcd don't have DC pin - how to connect?

### DISCUSSION THREAD

**Discussion by rozrabiak** [Bodmer/TFT_eSPI#3153] [Q&A - General]
Hi!
I need help with connection ssd1963 lcd to esp32/arduino.
My ssd1963 don't have DC pin (for TFT_DC).
How to connect that lcd?
[EDIT] - is that library compatible with Chip ESP32-D0WDQ6?
Thank's!

![IMG_20240202_001253](https://github.com/Bodmer/TFT_eSPI/assets/73905882/54b54b05-00d9-46c5-b9fb-c71f3c9deb87)

**Comment by rozrabiak** [comment]
Ok, RS pin is DC.
I connected all wires, checked config but only backlight work's..

**Comment by rozrabiak** [comment]
Ok, my program works, but I think - my lcd was corrupted.
Look at this video to the end.
This is example code from library.
After 30 seconds screen was greyed out and only backlight works.

[https://photos.app.goo.gl/k1vLx8RastqjNtFP8](url)

**Reply by drompus** [reply]
Have you managed to fix it?

**Comment by Vutoan6188** [comment]
You can see this video for answer

https://m.youtube.com/watch?v=ZjPZcyIQFhw&t=435s
